source/Body.cpp

```diff
@@ -33,8 +34,10 @@ using namespace std;
 
 // Constructor, based on a Sprite.
 Body::Body(const Sprite *sprite, Point position, Point velocity, Angle facing, double zoom)
-	: position(position), velocity(velocity), angle(facing), zoom(zoom), sprite(sprite), randomize(true)
+	: position(position), velocity(velocity), zoom(zoom), sprite(sprite), randomize(true)
 {
+	position += center;
+	Turn(facing);
```

Comment:
Actually, I see why this didn't work: this constructor doesn't get its `center` initialized. The center is only initialized by the LoadSprite function.